ietf/doc/views_doc.py

```diff
@@ -603,12 +603,7 @@ def document_main(request, name, rev=None, document_html=False):
         additional_urls = doc.documenturl_set.exclude(tag_id='auth48')
 
         # Stream description and name passing test
-        if doc.stream != None:
-            stream_desc = doc.stream.desc
-            stream = "draft-stream-" + doc.stream.slug
-        else:
-            stream_desc = "(None)"
-            stream = "(None)"
+        stream = ("draft-stream-" + doc.stream.slug) if doc.stream != None else "(None)"
```

Comment:
nit: "is not None" instead of "!= None"

Reply:
There are seven "!= None" and four "is not None" in the file.  Let me know if you want to standardize on one.

Reply:
Should definitely be `is not None` (citation: https://peps.python.org/pep-0008/#programming-recommendations)

We're not obsessively zealous about it, but I try to rephrase things as I encounter them and it'd be great if you don't mind cleaning these up!